app/controllers/users/articles_controller.rb

```diff
@@ -20,11 +20,31 @@ def authorize_resource!
     end
 
     def authorized_articles
-      authorized(Article.all)
+      authorized(Article.all.includes(:comments))
     end
 
     def article_params
       params.require(:article).permit(:title, :content).merge(user: current_user)
     end
+
+    def fetch_article
+      if params[:id].present?
+        Article.find(params[:id])
+      elsif current_company.present?
+        fetch_company_article
+      elsif params[:article].present?
+        Article.new(article_params)
+      else
+        Article.new
```

Comment:
is it normal that article has not user in this case?

Reply:
No, thank you! :)